packages/server/src/api/controllers/public/mapping/applications.ts

```diff
@@ -0,0 +1,32 @@
+import { Application, ApplicationOutput } from "./types"
```

Comment:
It could be nice if these were inverted, where `Application` is the full resource (the output) and the input one becomes `ApplicationInput`